app/Main.hs

```diff
@@ -79,6 +82,7 @@ graphOptions = GraphOptions
         allPathsCmd = AllPaths <$> argument str (metavar "module from") <*> argument str (metavar "module to")
         csvCmd = ToCSV <$> switch (long "no-header" <> short 'n' <> help "Omit CSV header")
         catCmd = Cat <$> some1 (argument str (metavar "graph.json"))
+        removeCmd = Remove <$> switch (short 'r' <> help "Use regex") <*> some1 (argument str (metavar "module"))
```

Comment:
I was tempted to use an arg scheme similar to grep..but `r` for `regex` seems fine.